app/helpers/agent_helper.rb

```
@@ -100,18 +100,47 @@ def agent_identifier_input(index, name_prefix, value = '', is_organization: true
+
+      if link
+        link_to(value, target: '_blank', rel: 'noopener noreferrer', title: "#{schema_agency}: #{notation}") do
+          content
+        end
+      else
+        content_tag(:span, title: "#{schema_agency}: #{notation}") do
+          content
+        end
+      end
+    end.join(' ').html_safe
   end
```

RuboCop (1.73)

[convention] Assignment Branch Condition size for display_identifiers is too high. [<11, 20, 9> 24.54/23]

(Metrics/AbcSize)